common/core/desktop/tests/unit/kmnkbd/test_kmx_xstring.cpp

```diff
@@ -21,14 +21,47 @@ using namespace std;
 #define U_UC_SURROGATE u"\U0001F609"
```

Comment:
```suggestion
#define U_UC_SURROGATE u"\u0001F609"
```

To be consistent with all the other constants using `\u`.